src/vm/types/program.zig

```diff
@@ -175,6 +175,16 @@ pub const Program = struct {
         }
         return parsed_data;
     }
+
+    pub fn readBuiltins(self: Self, allocator: Allocator) !std.ArrayList([]const u8) {
```

Comment:
you can access the `builtins` field directly off parsed_json.value.builtins and it will be a list of strings